tests/unit/integrations_kafka_consumer.cpp

```diff
@@ -178,13 +178,7 @@ TEST_F(ConsumerTest, BatchInterval) {
 TEST_F(ConsumerTest, StartStop) {
   Consumer consumer{CreateDefaultConsumerInfo(), kDummyConsumerFunction};
 
-  auto start = [&consumer](const bool use_conditional) {
-    if (use_conditional) {
-      consumer.StartIfStopped();
-    } else {
-      consumer.Start();
-    }
-  };
+  auto start = [&consumer]() { consumer.Start(); };
```

Comment:
It might make sense to fully remove this lambda and just use `consumer.Start()` instead.